src/chips/nrf51822/gpio.rs

```diff
@@ -101,3 +234,15 @@ pub static mut PA : Port = Port {
         GPIOPin::new(28), GPIOPin::new(29), GPIOPin::new(30), GPIOPin::new(31),
     ],
 };
+
+#[no_mangle]
+#[allow(non_snake_case)]
+pub unsafe extern fn GPIOTE_Handler() {
```

Comment:
OK, we'll now have to port `generic_isr` from the cortex-m4 crate to the cortex-m0 (should be straight forward) and use the same trick, but that's not for this pull request (and doesn't actually affect this code)